emigration.lua

```diff
@@ -210,8 +219,9 @@ dfhack.onStateChange[GLOBAL_KEY] = function(sc)
         return
     end
 
-    local persisted_data = dfhack.persistent.getSiteData(GLOBAL_KEY, {enabled=false})
+    local persisted_data = dfhack.persistent.getSiteData(GLOBAL_KEY, {enabled=false,last_cycle_tick=0})
```

Comment:
this will leave `last_cycle_tick` as `nil` in current games with saved state. It would be better to normalize the value here so you can remove the check for `nil` in `event_loop`.

I suggest a pattern like this:

```lua
    state = get_default_state()
    utils.assign(state, dfhack.persistent.getSiteData(GLOBAL_KEY, state))
```